src/storage/mod.rs

```diff
@@ -260,11 +279,13 @@ impl<'e, T, D> Storage<'e, T, D>
                    "Attempt to get an unchecked entry from a storage: {:?} {:?}",
                    entry.original,
                    self as *const Storage<'e, T, D>);
+
         unsafe { self.data.inner.get(entry.id) }
     }
 }
 
-/// The status of an insert operation
+/// The status of an `insert()`ion into a storage operation.
```

Comment:
In the documentations I have seen (including standard library) function are written without parentheses (in this case \`insert\`).